examples/stable/test/end_to_end/candid_rpc/candid_backtraces/benchmarks.md

```diff
@@ -0,0 +1,34 @@
+⚠️ **WARNING: Benchmark process failed for version 0.30.0**
```

Comment:
Get rid of this benchmark file, do not run the benchmarks and do not commit anything to do with benchmarks.